### ISSUE TITLE
[Linux Arm64] crash on Skiko 0.7.52

### ISSUE THREAD

**Issue by asapha** [JetBrains/skiko#672]
There's no crash when running
`./gradlew :SkiaAwtSample:run -Pskiko.version=0.7.51`

But running 
`./gradlew :SkiaAwtSample:run -Pskiko.version=0.7.52`
results in 
```java
Exception in thread "AWT-EventQueue-0" java.lang.UnsatisfiedLinkError: /home/parallels/.skiko/dc8a7eb9738c388587dde337c64f21082a7c32c9c56c84b8549a2a36186fdfc7/libskiko-linux-arm64.so: /home/parallels/.skiko/dc8a7eb9738c388587dde337c64f21082a7c32c9c56c84b8549a2a36186fdfc7/libskiko-linux-arm64.so: undefined symbol: _ZNSt19_Sp_make_shared_tag5_S_eqERKSt9type_info
        at java.base/jdk.internal.loader.NativeLibraries.load(Native Method)
        at java.base/jdk.internal.loader.NativeLibraries$NativeLibraryImpl.open(NativeLibraries.java:388)
        at java.base/jdk.internal.loader.NativeLibraries.loadLibrary(NativeLibraries.java:232)
        at java.base/jdk.internal.loader.NativeLibraries.loadLibrary(NativeLibraries.java:174)
        at java.base/java.lang.ClassLoader.loadLibrary(ClassLoader.java:2389)
        at java.base/java.lang.Runtime.load0(Runtime.java:755)
        at java.base/java.lang.System.load(System.java:1953)
        at org.jetbrains.skiko.Library.loadLibraryOrCopy(Library.kt:20)
        at org.jetbrains.skiko.Library.findAndLoad(Library.kt:113)
        at org.jetbrains.skiko.Library.load(Library.kt:59)
        at org.jetbrains.skiko.SkiaLayer.<clinit>(SkiaLayer.awt.kt:32)
        at SkiaAwtSample.AppKt.createWindow$lambda$1(App.kt:35)
        at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
        at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:771)
        at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
        at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:716)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
        at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
        at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:741)
        at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
        at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
        at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
        at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
        at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
        at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

Tested on an Ubuntu VM (22.04) via Parallels on a M1 Mac.

`uname -a`
> Linux ubuntu-linux-22-04-desktop 5.19.0-35-generic #36-Ubuntu SMP PREEMPT_DYNAMIC Fri Feb 3 18:36:47 UTC 2023 aarch64 aarch64 aarch64 GNU/Linux

**Comment by dima-avdeev-jb** [comment]
Thanks!
This bug is reproduced with JDK built in IntelliJ IDEA Community and only with skiko version 0.7.52.
And not reproduced with OpenJDK "11.0.18" 2023-01-17.

I am trying to build skiko from the same commit to mavenLocal, but it doesn't reproduced.

**Comment by dima-avdeev-jb** [comment]
Fixed in version 0.7.54

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/SKIKO-672">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
